specdb/build/privatedb.py

```diff
@@ -72,12 +75,19 @@ def grab_files(tree_root, skip_files=('c.fits', 'C.fits', 'e.fits',
                     if not os.path.isfile(ofile):
                         print("{:s} not present".format(ofile))
                 if flg:
-                    # import pdb;pdb.set_trace()
                     pfiles.append(ofile)
         # walk
         folders = next(walk)[1]
+    # Grab meta file (if one exists)
+    mfile = glob.glob(tree_root+'/*_meta.json')
+    if len(mfile) == 1:
+        mfile = mfile[0]
+    elif len(mfile) == 0:
+        mfile = None
+    else:
+        raise IOError("Multiple meta files in branch {:s}.  Limit to one".format(tree_root))
     # Return
-    return pfiles
+    return pfiles, mfile
```

Comment:
For some reason there is duplication of pfiles (this was an issue before this PR anyways).